tests/test_multilingual_local_recall.py

```
@@ -16,6 +23,130 @@ def test_recall_tokens_preserve_unicode_words():
+        "orion-telemetrie atlas-cache",
+    )
+    assert score == 1.0
+
+
+@pytest.mark.parametrize(
+    "content",
+    (
+        "The orion_telemetrie_api is healthy.",
+        "The orion.telemetrie.api is healthy.",
+        "The orion/telemetrie/api is healthy.",
+    ),
```

Ruff (0.15.21)

[warning] Wrong values type in `pytest.mark.parametrize` expected `list`

Use `list` for parameter values

(PT007)